crypto/store/loader_file.c

```diff
@@ -10,6 +10,7 @@
 #include <string.h>
 #include <sys/stat.h>
 #include <assert.h>
+#include <ctype.h>
```

Comment:
Are this PR supposed to remove this header file inclusion (at least from libcrypto) ?